Update _index.md for catalog curation (#25775)
DataDog/documentation: content/en/service_catalog/customize/_index.md

If you are using APM and would like to remove the automatically named *services* from your Service Catalog and Service Map, you can opt in to new [inferred entities experience][7], which allows you to filter Service Catalog entries by entity type, such as database, queue, or third-party dependencies. You can optionally [remove][8] any [service overrides][9] like service:my-service-http-client from your catalog or map. 

## Enrich auto-detected services with metadata 

> To specify on-call, source code, or documentation for your services, you can add metadata to any existing services via the UI, APIs, or [other automation][10]. 2.2 is the recommended version. To try experimental features like enhanced relationship mapping and precise code location, you can opt into the beta program for [schema 3.0][3] by [submitting a request][4].

Reviewer: ```suggestion
To specify on-call, source code, or documentation for your services, you can add metadata to any existing services using the UI, APIs, or [other automation][10]. 2.2 is the recommended version. To try experimental features like enhanced relationship mapping and precise code location, opt in to the beta program for [schema 3.0][3] by [submitting a request][4].
```
